src/BeerXMLElement.h

```diff
@@ -57,7 +58,7 @@ class BeerXMLElement : public QObject
 
    friend class Database;
 public:
-   BeerXMLElement(Brewtarget::DBTable table, int key);
+   BeerXMLElement(Brewtarget::DBTable table, int key, QString t_name = QString(), bool t_display = false, QString folder = QString());
```

Comment:
The line is really long. 
May i be an annoying bastard and ask you to split it into several lines?
It is not required but it would probably help with the readability

Reply:
I will work on the formatting.

If you are getting prompted for a password, I will promise your dbType is 2 (postgres) right now, and it sounds like you want it to be 1 (sqlite). That was one of the more uncomfortable changes I made, and it seems it didn't work as expected.

The quick fix will be to add config_version=1 and set dbType=1 in the [General] section of the config. Obviously, I am not suggesting we ship the code like this. I will fix it and figure out what broke.